docs/_docs/windows.md

```diff
@@ -129,13 +129,12 @@ choco install libxslt -Source "https://www.nuget.org/api/v2/"
 
 choco install libiconv -Source "https://www.nuget.org/api/v2/
 
-gem install nokogiri --^
-   --with-xml2-include=C:\Chocolatey\lib\libxml2.2.7.8.7\build\native\include^
-   --with-xml2-lib=C:\Chocolatey\lib\libxml2.redist.2.7.8.7\build\native\bin\v110\x64\Release\dynamic\cdecl^
-   --with-iconv-include=C:\Chocolatey\lib\libiconv.1.14.0.11\build\native\include^
-   --with-iconv-lib=C:\Chocolatey\lib\libiconv.redist.1.14.0.11\build\native\bin\v110\x64\Release\dynamic\cdecl^
-   --with-xslt-include=C:\Chocolatey\lib\libxslt.1.1.28.0\build\native\include^
-   --with-xslt-lib=C:\Chocolatey\lib\libxslt.redist.1.1.28.0\build\native\bin\v110\x64\Release\dynamic
+gem install nokogiri -- --with-xml2-include=C:\Chocolatey\lib\libxml2.2.7.8.7\build\native\include
```

Comment:
These are nice but could be split to a separate PR. Or you could change the snippet's highlighting to `bat` and leave the lines as they where. `^` is the right way to continue a line in Windows batch files.